egs2/TEMPLATE/svs1/README.md

```diff
@@ -0,0 +1,442 @@
+Make your config.
+```yaml
+# e.g.
+svs_conf:
+    langs: 4   # Number of languages
+    spks: 5    # Number of speakers
+```
+Please run the training from stage 6.
+```sh
+$ ./run.sh --stage 6 --use_lid true --use_sid true --train_config /path/to/your_multi_spk_multi_lang_config.yaml
+```
+
+
+### Vocoder training
+
+If your `--vocoder_file` is set to none, Griffin-Lim will be used.
+You can also train corresponding vocoder using [kan-bayashi/ParallelWaveGAN](https://github.com/kan-bayashi/ParallelWaveGAN)..
+
+Pretrained vocoder is like follows:
+
+```sh
+*_hifigan.v1 
+├── checkpoint-xxxxxxsteps.pkl
+├── config.yml
+└── stats.h5
+```
+
+```sh
+# Use the vocoder trained by `parallel_wavegan` repo manually
+$ ./run.sh --stage 7 --vocoder_file /path/to/checkpoint-xxxxxxsteps.pkl --inference_tag decode_with_my_vocoder
+```
+
+### Evaluation
+
+We provide four objective evaluation metrics:
+
+- Mel-cepstral distortion (MCD)
+- Logarithmic rooted mean square error of the fundamental frequency (F![1](http://latex.codecogs.com/svg.latex?_0)RMSE)
+- Semitone accuracy (Semitone ACC)
+- Voiced / unvoiced error rate (VUV_E)
+
+For MCD, we apply dynamic time-warping (DTW) to match the length difference between ground-truth singing and generated singing.
+
+Here we show the example command to calculate objective metrics:
+
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate MCD
+./pyscripts/utils/evaluate_mcd.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gt_wavdir_or_wavscp.scp
+```
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate log-F0 RMSE & Semitone ACC & VUV Error Rate
+./pyscripts/utils/evaluate_f0.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gen_wavdir_or_wavscp.scp
+```
+While these objective metrics can estimate the quality of synthesized singing, it is still difficult to fully determine human perceptual quality from these values, especially with high-fidelity generated singing.
+Therefore, we recommend performing the subjective evaluation (eg. MOS) if you want to check perceptual quality in detail.
+
+You can refer [this page](https://github.com/kan-bayashi/webMUSHRA/blob/master/HOW_TO_SETUP.md) to launch web-based subjective evaluation system with [webMUSHRA](https://github.com/audiolabs/webMUSHRA).
+
+## About data directory
+
+Each directory of training set, development set, and evaluation set, has same directory structure. See also http://kaldi-asr.org/doc/data_prep.html about Kaldi data structure. 
+We recommend you running `ofuton_p_utagoe_db` recipe and checking the contents of `data/` by yourself.
+
+```bash
+cd egs/ofuton_p_utagoe_db/svs1
+./run.sh
+```
+
+- Directory structure
+    ```
+    data/
+    ├── tr_no_dev/     # Training set directory
+    │   ├── text       # The transcription
+    │   ├── label      # Specifying start and end time of the transcription 
+    │   ├── midi.scp   # MIDI file path

```

Comment:
I think we should re-design the data format, especially for label file, which is difficult to read or analysis